grisera_api/time_series/time_series_service_graphdb_with_signal_values.py

```diff
@@ -0,0 +1,341 @@
+    TimestampNodesIn
+from time_series.time_series_service_graphdb import TimeSeriesServiceGraphDB
+
+
+class TimeSeriesServiceGraphDBWithSignalValues(TimeSeriesServiceGraphDB):
+
+    def save_time_series(self, time_series: TimeSeriesIn):
+        """
+        Send request to graph api to create new time series
+        Args:
+            time_series (TimeSeriesIn): Time series to be added
+        Returns:
+            Result of request as time series object
+        """
+        result = super().save_time_series(time_series)
+        if result.errors is not None:
+            return result
+
+        time_series_id = result.id
+
+        errors = self.save_signal_values(time_series.signal_values, time_series_id,
+                                         self.get_experiment_id(time_series_id), time_series.type)
+        if errors is not None:
+            return TimeSeriesOut(**time_series.dict(), errors=errors)
+
+        result.signal_values = self.get_signal_values(time_series_id, time_series.type)
+
+        return result
+
+    def get_experiment_id(self, time_series_id: int):
+        query = {
+            "nodes": [
+                {
+                    "id": time_series_id,
+                    "label": "Time Series"
+                },
+                {
+                    "label": "Observable Information"
+                },
+                {
+                    "label": "Recording"
+                },
+                {
+                    "label": "Participation"
+                },
+                {
+                    "label": "Activity Execution"
+                },
+                {
+                    "label": "Activity Execution"
+                },
+                {
+                    "label": "Experiment",
+                    "result": True
+                }
+            ],
+            "relations": [
+                {
+                    "begin_node_index": 0,
+                    "end_node_index": 1,
+                    "label": "hasObservableInformation"
+                },
+                {
+                    "begin_node_index": 1,
+                    "end_node_index": 2,
+                    "label": "hasRecording"
+                },
+                {
+                    "begin_node_index": 2,
+                    "end_node_index": 3,
+                    "label": "hasParticipation"
+                },
+                {
+                    "begin_node_index": 3,
+                    "end_node_index": 4,
+                    "label": "hasActivityExecution"
+                },
+                {
+                    "begin_node_index": 5,
+                    "end_node_index": 4,
+                    "label": "nextActivityExecution",
+                    "min_count": 0
+                },
+                {
+                    "begin_node_index": 6,
+                    "end_node_index": 5,
+                    "label": "hasScenario"
+                },
+            ]
+        }
```

Comment:
Is this an universal query to get time series?

Reply:
This is query to get experiment id from time series id. I need this to make common timestamp line connected to the experiment.